app/helpers/filters_helper.rb

```diff
@@ -165,9 +165,9 @@ def filters_applied_text(filter_type)
     applied_filters_count(filter_type).zero? ? "No filters applied" : "#{pluralize(applied_filters_count(filter_type), 'filter')} applied"
   end
 
-  def reset_filters_link(filter_type, path_params = {})
+  def reset_filters_link(filter_type, filter_path_params = {})
     if applied_filters_count(filter_type).positive?
-      govuk_link_to "Clear", clear_filters_path(filter_type:, path_params:)
+      govuk_link_to "Clear", clear_filters_path(filter_type:, filter_path_params:)
```

Comment:
Path_params kept on getting cleared, I think because of this: https://github.com/rails/rails/pull/43770